### ISSUE TITLE
connection failure for gh0stp0st registry

### ISSUE THREAD

**Issue by CPCer** [SalmonProject/SalmonEmailToREST#1]
closed

**Comment by CPCer** [comment]
@SalmonProject

**Comment by CPCer** [comment]
And it constantly pops up a DEBUG notice.... @SalmonProject

**Comment by SalmonProject** [comment]
I am not surprised it's unblocked - I think right now it could get 1 or 2 more orders of magnitude users before the government would even realize it existed!

About the post distribution: they will see that it was you who reported the deleted post. However, that "see that it is you" is entirely within the GhostPost system. GhostPost never posts/reblogs anything on Weibo. That might be a point worth more clearly stating somewhere: I can imagine people not wanting that to happen, and I can also imagine people seeing how GhostPost presents posts and wondering if it's interacting with weibo.com in some way.

> DEBUG notice

Unless you had already encountered that problem several days before your post, that should be because the server was down. Sorry, it's back up with the problem fixed - I will be monitoring it more closely!

**Comment by SalmonProject** [comment]
...although now that I think of it, the server going down is not a good reason to constantly make popups. (I am guessing it averaged about 1 every 10 minutes?) I'll open a new bug about that...

**Comment by CPCer** [comment]
it now works!
Yes after i logged in it hasn't been poping up till now. still in use

**Comment by CPCer** [comment]
Actually, there are already Chinese reports and discussions about Ghostpost, and i am almost sure those who work for the Communist Party have noticed. So...Hope the best. we can still use various VPNs to use Ghostpost.

**Comment by SalmonProject** [comment]
Wow, really? Maybe I have gotten too used to things being quiet... I will keep a close an eye on the server for sure.

**Comment by CPCer** [comment]
![image](https://cloud.githubusercontent.com/assets/8787958/17798754/e2761c2c-6606-11e6-8201-610562a35993.png)
so what's this mean? I'm trying to find a "deleted" post when this shows

**Comment by CPCer** [comment]
i mean do you have a contact on Skype, Hangouts, or Tele or What's...(can't believe i have so many...)I'd like to talk with you privately...or maybe about L8n

**Comment by SalmonProject** [comment]
Oh! That was not supposed to be in there for the release. I have fixed it - https://salmon.cs.illinois.edu/files/GhostPost.crx is now an updated version that won't do that anymore.

Give fed2@case.edu on Hangouts a try (it's a Google Apps account). I should usually be available on there.

**Comment by CPCer** [comment]
Sadly, i met with this problem...
http://stackoverflow.com/questions/21475701/cant-install-my-custom-chrome-extension-crx-magic-number-invalid
and the "Load unpacked extension..." works.
BTW, I've sent invitations to your Hangouts :).

**Comment by CPCer** [comment]
sorry to interrupt you once and again... I just found this cool extension so amazing and wanna do even more with it...May i know your idea about this Ghostpost, whether it is something out of interest or a formal project...
BTW I've sent an invitation on Hangouts... If possible, maybe email address (which might be used more frequently by you?)is better,,,

**Comment by SalmonProject** [comment]
Oh no, I'm sorry I left this for this long! I would absolutely like to advance GhostPost and make it useful. I guess it is halfway between "my personal interest" and "format project": I am working on it as part of a PhD program, but I am essentially alone on it. There is certainly work to be done if it picks up substantial amounts of users.

Why don't you email me at fed2@illinois.edu. I have not seen anything on Hangouts - why don't you email me from a Google account you want to use with Hangouts, and I will try to get in touch with you (fed2@case.edu and/or a traditional @gmail.com account).
